app.js

```diff
@@ -29,36 +29,45 @@ function checkResponseNo(response) {
     console.log('User response: ' + response + '. Wrong!');
   }
 }
-
-userResponse = prompt('Does Kyle like rain drops on roses?');
-console.log('Does Kyle like rain drops on roses?');
-checkResponseYes(userResponse);
-numQuestions++;
-
+function questionOne(){
```

Comment:
indentation and white-space are consistent, very good for readability. Nice job!